test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-shards-test.yml

```diff
@@ -0,0 +1,5 @@
+gcloud:
+  app: test_runner/src/test/kotlin/ftl/fixtures/tmp/apk/app-debug.apk
+  test: test_runner/src/test/kotlin/ftl/fixtures/tmp/apk/app-multiple-success-debug-androidTest.apk
+  test-targets-for-shard:
+    - com.example.test_app.bar.BarInstrumentedTest
```

Comment:
I think there is `class` missing. 
We could also add another shard. IMO it will be more realistic
```
- class com.example.test_app.foo.FooInstrumentedTest
```

Reply:
hmm will look into it.

Reply:
Yes class is missing, will add another class to as a reference test.